server/startrabbit.sh

```diff
@@ -17,9 +17,16 @@ else
 			rabbitmqctl join_cluster --ram rabbit@$CLUSTER_WITH
 		fi
 		rabbitmqctl start_app
+                
+        if [ "$RABBITMQ_DEFAULT_USER" -a "$RABBITMQ_DEFAULT_PASS"] then
```

Comment:
This statement `"$RABBITMQ_DEFAULT_USER" -a "$RABBITMQ_DEFAULT_PASS"` feels wrong, bash complained about this too.